pkg/controller/nodeipam/ipam/cidr_allocator.go

```diff
@@ -118,7 +119,8 @@ func New(kubeClient clientset.Interface, cloud cloudprovider.Interface, nodeInfo
 	case RangeAllocatorType:
 		return NewCIDRRangeAllocator(kubeClient, nodeInformer, allocatorParams, nodeList)
 	case CloudAllocatorType:
-		return NewCloudCIDRAllocator(kubeClient, cloud, nwInformer, gnpInformer, nodeInformer, allocatorParams)
+		eventBroadcaster := record.NewBroadcaster()
```

Comment:
why this change?

Reply:
Responded in https://github.com/kubernetes/cloud-provider-gcp/pull/656#discussion_r1539013915